src/js/components/Search.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Search = (/* props */) => (
```

Comment:
I can change the name. The skeleton is for the other team members to work with (implement search MUI text field; use Bloodhound.js - possibly - or roll our own filters ... that sort of thing).

Sorry, I need to be clear in the comments.